.buildkite/release-builds.yml

```diff
@@ -11,10 +11,18 @@ steps:
     command: ".buildkite/commands/release-build-wordpress.sh $BETA_RELEASE"
     plugins: [$CI_TOOLKIT_PLUGIN]
     notify:
-    - slack: "#build-and-ship"
+      - slack: "#build-and-ship"
```

Comment:
On seeing this I wondered "How has it worked all this time?!" but it turns out YAML (or at least the Ruby parser and Buildkite's parser) are flexible on array indentation:

![image](https://github.com/user-attachments/assets/a735c58a-fb8c-43ab-9091-ecbc87804b53)

Reply:
Yep, I was also surprised 🙂

Reply:
Was surprised as well 😅 

💡  Tip of the day: If you want to test how `buildkite-agent` itself would interpret your YAML (and thus be sure to rely on its implementation of YAML parser), you can use `buildkite-agent pipeline upload --dry-run` to print its interpretation of the pipeline. It prints the interpreted pipeline to stdout—in JSON by default, but you can use `--format=yaml` if you prefer YAML output that would be closer to the input file you provided.

For example, using a pipe to pass the YML to parse:
```bash
# Note: you need to provide the `-i` flag (but not add the `-t` flag!) for `docker run` to be able to read from STDIN through the pipe properly
echo "notify:\n- x" | docker run --rm -i buildkite/agent pipeline upload --dry-run --format=yaml
cat .buildkite/release-builds.yml | docker run --rm -i buildkite/agent pipeline upload --dry-run --format=yaml
```
Or if you want to pass it a file path to parse:
```bash
docker run --rm -v ./:/app -w /app buildkite/agent pipeline upload --dry-run --format=yaml .buildkite/pipeline.yml
```